src/main/java/htsjdk/tribble/index/Index.java

```diff
@@ -73,41 +73,42 @@
     /**
      * Writes the index into a file.
      *
+     * Default implementation delegates in {@link #write(Path)}
```

Comment:
Sorry to nitpick the language, but delegates **to** instead of delegates **in** is more idiomatic.